src/tqec/circuit/generation_test.py

```diff
@@ -32,9 +32,9 @@ def _expected_circuit() -> stim.Circuit:
     R 0
     TICK
     TICK
-    TICK
     CX 1 0
     TICK
+    TICK
```

Comment:
Why did that change? Was there a bug in the previous implementation?

Reply:
Previously, I misuse the `x_boundary_orientation`. In all codes related to that, I considered it as the "logical X operator orientation" in my mental model. So the cnot ordering was actually wrong. But as I also wrongly parsed the orientation from `CubeType`, the effect canceled out and gave the correct results. Now I fixed the CNOT ordering as well as the orientation parsing part.

Reply:
Ok, I just re-checked the `CX` ordering in CSS `ZZ` stabilizer measurement on the left boundary and the updated code seems correct.